snmplib/snmp-tc.c

```diff
@@ -255,7 +255,7 @@ ctime_to_timet(const char *str)
      */
 
 #ifdef HAVE_STRUCT_TM_TM_ISDST
-#if HAVE_DECL_DAYLIGHT
+#if HAVE_DECL_DAYLIGHT==1
```

Comment:
This change seems incorrect to me. I'm concerned this change will cause a compiler error if HAVE_DECL_DAYLIGHT is not defined.

Reply:
It seems that the AC_CHECK_DECL will always #define this as either 0 or 1. I originally had changed it into a #ifdef but that caused errors. It has compiled on various Linux and *BSD systems like this.

Reply:
If HAVE_DECL_DAYLIGHT is always 0 or 1, does this mean that the above change is not necessary?

Reply:
In principle, yes. I did it as a kind of warning to my self - and others - to not change that #if into an #ifdef, because that will fail.